### ISSUE TITLE
VOTE: Release RO-Crate 1.1.0 RC1

### ISSUE THREAD

**Issue by stain** [ResearchObject/ro-crate#109]
I am pleased to be calling this vote for the release of
**RO-Crate specification 1.1** 

Votes are open until [Monday 2020-10-26 17:00 UTC](https://everytimezone.com/s/11ad82e9).

The release candidate to be voted over is available at:

* HTML: <https://stain.github.io/ro-crate/1.1-RC1/index.html> 
 (to become `https://w3id.org/ro/crate/1.1` redirecting to `https://www.researchobject.org/ro-crate/1.1/`)
* Single page HTML: <https://github.com/ResearchObject/ro-crate/releases/download/1.1.0-RC1/ro-crate-1.1.0-RC1.html>
* Single page PDF: <https://github.com/ResearchObject/ro-crate/releases/download/1.1.0-RC1/ro-crate-1.1.0-RC1.pdf>
* JSON-LD context: <https://stain.github.io/ro-crate/1.1-RC1/context.jsonld> 
 (to become `https://w3id.org/ro/crate/1.1/context` redirecting to `https://www.researchobject.org/ro-crate/1.1/context.jsonld`)
* RO-Crate: <https://stain.github.io/ro-crate/1.1-RC1/ro-crate-preview.html> <https://stain.github.io/ro-crate/1.1-RC1/ro-crate-metadata.json>
* Changelog and downloads: https://github.com/ResearchObject/ro-crate/releases/tag/1.1.0-RC1

Please vote and/or comment on any bugs on this github issue #109 **before Monday** 2020-10-26 17:00 UTC. (_You can use "GitHub reaction" to vote_)

Anyone can participate in testing and voting, not just [RO-Crate community](https://www.researchobject.org/ro-crate/community.html) members, please feel free to check the release candidate and provide your votes.

tag: https://github.com/ResearchObject/ro-crate/releases/tag/1.1.0-RC1
commit: c9aa82ad2a7689907f16aaa0ea4f7e517a065a9a

**Comment by stain** [comment]
I'm also preparing the Zenodo DOI with the release PDF/HTML. 

**Author list**: This is simply the https://www.researchobject.org/ro-crate/community members at time of release.

Note: @simleo just joined the team, but I had not updated the file in time for this RC1. Will include in final author list.

We should probably also acknowledge contributions in discussions from @marcolarosa although he did not sign up on #1 - anyone else?

**Comment by dgarijo** [comment]
@stain,
I skimmed through the pdf document and detected a few inconsistencies (nothing major), which I annotated in the attached pdf. Apart from that, +1 from me. Summary: 
- The status: Recommendation is duplicated.
- DOI still does not resolve (you may not have prepared it yet)
- Inconsistent capitalization os Schema.org (sometimes as schema.org; other as Schema.org).
- Sometimes the document will use "keys" to refer to "properties", which I think it may be confusing
- In sec 5.4 the heading of the section describes 3 levels of assertions, but only 2 bullet points are provided.
- In some cases there is weird capitalization (e.g., Associated Publications instead of Associated publications, Use Case instead of use case, etc.)
- Section 6.1.2 describes a Datacite DOI, but that term (Datacite) has not been used before. Why not just say DOI?
- I also had some very small comments (can be found in the pdf). Suggestions like using "the Linked Data Principles" instead of "Linked Principles" (as if there was other set of principles available). 
[ro-crate-1.1.0-RC1-DG.pdf](https://github.com/ResearchObject/ro-crate/files/5432575/ro-crate-1.1.0-RC1-DG.pdf)

**Comment by jmfernandez** [comment]
Hi @stain , I think I found a typo in [Additional Metadata Standards](https://stain.github.io/ro-crate/1.1-RC1/metadata.html#additional-metadata-standards) . There `RepositoryFile` is mapped to https://pcdm.org/2016/04/18/models#Collection instead of https://pcdm.org/2016/04/18/models#File .

**Comment by jmfernandez** [comment]
I have also a question. When an RO-Crate is intertwined with a BagIt, the optional `fetch.txt` from BagIt describes the list of remote contents, as they could be or not in the BagIt. When an RO-Crate is not intertwined with a BagIt, which is the best way to tell that content like

```json
  {
    "@id": "survey-responses-2019.csv",
    "@type": "File",
    "encodingFormat": "text/csv",
    "url": "http://example.com/downloads/2019/survey-responses-2019.csv",
    "subjectOf": {"@id": "http://example.com/reports/2019/annual-survey.html"}
  },
```

may not be included?

**Comment by ptsefton** [comment]
In 4.2. The bullet points lack some indenting, and there's a missing bracket. Should look like this:
```
ro-crate-preview.html SHOULD:
-  Display at least the metadata relating to the Root Data Enity as static HTML without the need for scripting. It MAY contain extra features enabled by JavaScript.
-  When a Data Entity or Contextual Entity is referenced by its ID:
    -  If it has a name property, provide a link to its HTML version.
    -  If it does not have a name (e.g. a GeoCoordinates location), show it embedded in the HTML for the entity.
    - For external URI values, provide a link.
- For keys that resolve in the RO-Crate JSON-LD Context to a URI, indicate this (the simplest way is to link the key to its definition).
- If there is sufficient metadata, contain a prominent “Cite-as” text with a natural language data citation (see for example the FORCE11 Data Citation Principles).
-  If there are additional resources necessary to render the preview (e.g. CSS, JSON, HTML), link to them in a subdirectory ro-crate-preview-files/
```

**Comment by ptsefton** [comment]
In 5.3 reference to PCDM has two closing brackets:  (PCDM))

**Comment by ptsefton** [comment]
In the single page 6.2 the final two paragraphs are in a <pre class='note'> - rendering as raw markdown

**Comment by ptsefton** [comment]
IN 7.3, in the example I suggest that the File entities should have a name rather than a description (if there's only one, name is more useful.

```
{
    "@id": "survey-responses-2019.csv",
    "@type": "File",
    "contentSize": "26452",
    "name": "Survey responses",
    "encodingFormat": "text/csv"
  },
  {
    "@id": "https://zenodo.org/record/3541888/files/ro-crate-1.0.0.pdf",
    "@type": "File",
    "contentSize": "310691",
    "name": "RO-Crate specification",
    "encodingFormat": "application/pdf"
  }

```

**Comment by ptsefton** [comment]
LIkewise in 7.3.1 - external references should have names so a human-readable HTML file can be built.

**Comment by ptsefton** [comment]
In the single page HTML <pre class="tip"> and <pre class="note"> are used - these end up with raw markdown on screen.

**Comment by ptsefton** [comment]
In 9.4 the PCDM terms are shown in <del> tags - they should not be.

**Comment by ptsefton** [comment]
IN 10.3 - this example should have a name on the workflow sketch rather than a description as per the other examples:

```
{
  "@id": "workflow/workflow.svg",
  "@type": ["File", "ImageObject"],
  "encodingFormat":  ["image/svg+xml"],
  "name": "Diagram of RetroPath2.0 workflow",
  "about": {"@id": "workflow/workflow.knime"}
},


```

**Comment by ptsefton** [comment]
I reacted with a thumbs up to say that this is OK to release once typos and small suggestions like mine above are fixed. Let me know if you want a PR with my fixes.

**Comment by stain** [comment]
Thank you for all the feedback! I don't think we should need a new vote for the typo fixes - so agree with @ptsefton to consider vote+comment as a kind of conditional vote. Indicate if you think something in your comment could be delayed for later releases.

I'll try to go through some of these today from the top - put in a comment here if you volunteer to do a pull request on that change so we don't do it duplicate! As there were quite a few little things I don't think we'll push it out today; let's aim for fixing the typos by Thurs 2020-10-29 16:00 UTC so that we can push out a 🎃 Halloween release!

**Comment by stain** [comment]
@jmfernandez I think for external content which is not in `fetch.txt` then they should simply be [external web-based URIs](https://stain.github.io/ro-crate/1.1-RC1/data-entities.html#web-based-data-entities).  I would consider `fetch.txt` in BagIt as a way to _complete_ the Bag from some transfer mechanism - and as the RO-Crate lives inside `data/` and therefore it can reasonably assume it is in a completed bag. (Still the `url` is helpful to show origin).

What is unclear is what is the semantics in BagIt if a file exists, but is ALSO listed in `fetch.txt` - can you just delete that file from `data/` and expect to download it again later? Is the URL where it came from or just a mirror of it? 

jkunze/bagitspec#7 discusses `fetch.txt` semantics, and the conclusion is that it is kept for backwards compatibility, but will be removed in BagIt 2.0.

Well, luckily in BagIt you can then also compare the checksums in the `manifest-sha512.txt` file - if you download it again and it's the same checksum, then it's the same file. If checksums mismatch, then something changed - and we don't know who's fault that is.

**Comment by stain** [comment]
@ptsefton As for the `<pre class="tip">` boxes these are a bit of a [hack](https://jekyll-rtd-theme.rundocs.io/test/toasts.html) in https://rundocs.github.io/jekyll-rtd-theme/ - their non-rendering should only affect the single-HTML and single-PDF (which is made from single-HTML) but perhaps some `sed` magic could make them plain text `<div>` instead - if they also fix the closing `</pre>` that is!

**Comment by stain** [comment]
Fixed the tip-box with a little [pandoc filter](https://github.com/ResearchObject/ro-crate/blob/master/scripts/pandoc-admonition.py)

![notewarning](https://user-images.githubusercontent.com/253413/97616183-20639f80-1a14-11eb-9457-19a00d0562a7.png)

You won't easily see where the warning ends though, but almost all of them are single paragraphs.

**Comment by stain** [comment]
After merging fixes #111 #112 #113 and my own review #114  I've prepared 1.1.0-RC2 which is now ready for release.

You may want to quickly look over #114 changes.

https://github.com/ResearchObject/ro-crate/releases/tag/1.1.0-RC2
https://stain.github.io/ro-crate/1.1-RC2/about.html

"Fire in the hole!" - will push the release button later today Friday.  Thank you everyone, this is looking quite nice!
